utility/PLYExport.cpp

```diff
@@ -0,0 +1,105 @@
+#include "PLYExport.h"
+#include <fstream>
+
+bool PLYExporter::Export(const char *fname, const VF &tri, const VF &triNorm, const VF& triColor, const VI &tStart, const VI &tCount){
+    
+    std::ofstream f(fname);
+    if(!f.is_open())return false;
+    
+    // verifying equal sizes of individual containers makes the iteration simpler then
+    if( tri.size() != triNorm.size() ) return false;
+    if( tri.size() != triColor.size() ) return false;
+
+    // this part can be probably optimized...
+    size_t numOfTri = 0;
+    {
+        auto tci = tCount.cbegin();
+        for( ; tci != tCount.end(); ++tci ){
```

Comment:
Is there a reason you aren't just doing numOfTri+=stripVertexCount-2 for this loop? Just curious if there's a subtlety that I'm missing in the calculation.

Reply:
I already beat you to it :smile: